src/main/java/se/bjurr/gitchangelog/internal/git/GitRepo.java

```diff
@@ -142,6 +163,45 @@ public ObjectId getRef(final String fromRef) throws GitChangelogRepositoryExcept
     throw new GitChangelogRepositoryException(fromRef + " not found in:\n" + toString());
   }
 
+  public boolean hasSubmodules() {
+    return submodules != null && submodules.size() > 0;
```

Comment:
This null-check is not needed, if changing to empty map as default value.

Reply:
Fixed.